src/components/taskList/taskList.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../variables.scss';
+
+.taskList {
+  position: relative;
+  background: white;
+  padding: $base * 10;
+  margin: auto;
+  width: calc(100% - #{$base * 20});
+  height: 50%;
+
+  ul {
+    padding-left: $base * 4;
+    li {
+      padding: $base 0;
+      line-height: 1;
+      span {
+        visibility: hidden;
+        display: inline-block;
+        padding: $base $base 0 $base;
+        width: 16px;
+        height: 16px;
+        background-repeat: no-repeat;
+        background-position: $base $base;
+        &.delete {
+          background-image: url('../../images/cross.svg');
+        }
+        &.done {
+          background-image: url('../../images/checkMark.svg');
+        }
+        &.inprogress {
+          background-image: url('../../images/inprogress.svg');
```

Comment:
there is no such image in your repo, check it https://github.com/lexahnure/homeproject/tree/filter/src/images